plugins/worker/server/utils.py

```diff
@@ -134,9 +134,12 @@ def jobInfoSpec(job, token=None, logPrint=True):
     if isinstance(token, dict):
         token = token['_id']
 
+    apiUrl = getWorkerApiUrl()
+
     return {
         'method': 'PUT',
-        'url': '/'.join((getWorkerApiUrl(), 'job', str(job['_id']))),
+        'url': '/'.join((apiUrl, 'job', str(job['_id']))),
+        'apiUrl': apiUrl,
```

Comment:
I either have to be inconsistent with those, or inconsistent with the keys in this returning dictionary.